api/src/client/WebserviceLicenseStatusProcessorClient.test.ts

```diff
@@ -22,14 +23,15 @@ const entityWithAddress = (address: string): LicenseEntity => {
 describe("WebserviceLicenseStatusProcessorClient", () => {
   let stubLicenseStatusClient: jest.Mocked<LicenseStatusClient>;
   let searchLicenseStatus: SearchLicenseStatus;
+  const logger = DummyLogWriter;
```

Comment:
sand - can the variable name be updated to dummyLogger or mockLogger so it's clear what is being used